.github/workflows/export-dynamic.yaml

```diff
@@ -53,17 +73,18 @@ on:
         required: false
         type: boolean
         default: false
-  
+
 jobs:
-  export:    
+  export:
     runs-on: ubuntu-latest
 
     env:
       NODE_OPTIONS: --max-old-space-size=8192
-    
+      IMAGE_REGISTRY: ghcr.io/${{ github.repository_owner }}
```

Comment:
I think it should be a workflow call input.
Maybe also the user as registry user, and the password only would be passed as an environment variable name ?